homeassistant/components/deconz/climate.py

```diff
@@ -120,6 +144,31 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
 
         await self._device.async_set_config(data)
 
+    # Preset control
+
+    @property
+    def preset_mode(self) -> Optional[str]:
+        """Return preset mode."""
+        for hass_preset_mode, preset_mode in PRESET_MODES.items():
```

Comment:
We can make this more efficient by creating an inverted version of the `PRESET_MODES` dict constant that we look up here.

Reply:
Fixed in https://github.com/home-assistant/core/pull/43883